src/components/ListItem.jsx

```diff
@@ -1,12 +1,30 @@
 import './ListItem.css';
+import { deleteItem } from '../api/firebase.js';
+import { useRef } from 'react';
 
 export function ListItem({
 	name,
 	isDefaultChecked,
 	itemId,
 	setIsChecked,
 	setCheckedItemId,
+	listToken,
 }) {
+	const dialogRef = useRef();
+
+	function confirmDelete() {
+		dialogRef.current.showModal();
```

Comment:
Mind blown! Had no idea there was a built-in method that handles this sort of behavior so cleanly! Loved your original `alert` implementation but I dig this even more!

Reply:
Agree with everyone else here - this is a super cool!